Parse-Dashboard/app.js

```diff
@@ -87,21 +87,29 @@ module.exports = function(config, options) {
         newFeaturesInLatestVersion: newFeaturesInLatestVersion,
       };
 
-      //Based on advice from Doug Wilson here:
-      //https://github.com/expressjs/express/issues/2518
-      const requestIsLocal =
-        req.connection.remoteAddress === '127.0.0.1' ||
-        req.connection.remoteAddress === '::ffff:127.0.0.1' ||
-        req.connection.remoteAddress === '::1';
-      if (!options.dev && !requestIsLocal) {
-        if (!req.secure && !options.allowInsecureHTTP) {
+      for (const key in options) {
+        if (options[key] && !config[key]) {
```

Comment:
Does this work if an option has value `false`?

Reply:
Good thinking